packages/scaffold/src/views/w3m-upgrade-to-smart-account-view/index.ts

```diff
@@ -27,7 +28,11 @@ export class W3mUpgradeToSmartAccountView extends LitElement {
         .padding=${['0', '0', 'l', '0'] as const}
       >
         ${this.onboardingTemplate()} ${this.buttonsTemplate()}
-        <wui-link>
+        <wui-link
+          @click=${() => {
+            CoreHelperUtil.openHref('https://walletconnect.com/faq', '_blank')
```

Comment:
Could we create a `Navigation` unit/constant in `@common` and use it here like `Navigation.WEBSITE_FAQ` to avoid hard coded values? This would allow us to maintain all URLs or other sources/hard coded values in one place.